assembl/graphql/post.py

```diff
@@ -73,39 +78,41 @@ def resolve_creator(self, args, context, info):
-    body_entries = graphene.List(LangStringEntry, lang=graphene.String())
-    sentiment_counts = graphene.Field(SentimentCounts)
-    my_sentiment = graphene.Field(type=SentimentTypes)
-    indirect_idea_content_links = graphene.List(IdeaContentLink)
-    extracts = graphene.List(Extract)
-    parent_id = graphene.ID()
-    db_id = graphene.Int()
-    body_mime_type = graphene.String(required=True)
-    publication_state = graphene.Field(type=PublicationStates)
-    attachments = graphene.List(PostAttachment)
-    original_locale = graphene.String()
-    publishes_synthesis = graphene.Field(lambda: Synthesis)
+    creation_date = DateTime(description=docs.PostInterface.creation_date)
+    modification_date = DateTime(description=docs.PostInterface.modification_date)
+    subject = graphene.String(lang=graphene.String(), description=docs.PostInterface.subject)
+    body = graphene.String(lang=graphene.String(), description=docs.PostInterface.body)
+    subject_entries = graphene.List(LangStringEntry, lang=graphene.String(), description=docs.PostInterface.subject_entries)
+    body_entries = graphene.List(LangStringEntry, lang=graphene.String(), description=docs.PostInterface.body_entries)
+    sentiment_counts = graphene.Field(SentimentCounts, description=docs.PostInterface.sentiment_counts)
+    my_sentiment = graphene.Field(type=SentimentTypes, description=docs.PostInterface.my_sentiment)
+    indirect_idea_content_links = graphene.List(IdeaContentLink, description=docs.PostInterface.indirect_idea_content_links)
+    extracts = graphene.List(Extract, description=docs.PostInterface.extracts)
+    parent_id = graphene.ID(description=docs.PostInterface.parent_id)
+    db_id = graphene.Int(description=docs.PostInterface.db_id)
+    body_mime_type = graphene.String(required=True, description=docs.PostInterface.body_mime_type)
+    publication_state = graphene.Field(type=PublicationStates, description=docs.PostInterface.publication_state)
+    attachments = graphene.List(PostAttachment, description=docs.PostInterface.attachments)
+    original_locale = graphene.String(description=docs.PostInterface.original_locale)
+    publishes_synthesis = graphene.Field(lambda: Synthesis, description=docs.PostInterface.publishes_synthesis)
 
     def resolve_db_id(self, args, context, info):
         return self.id
 
     def resolve_extracts(self, args, context, info):
         return self.db.query(models.Extract
-            ).join(models.Content, models.Extract.content == self
-            ).options(joinedload(models.Extract.text_fragment_identifiers)
-            ).order_by(models.Extract.creation_date
-            ).all()
+                             ).join(models.Content, models.Extract.content == self
+                                    ).options(joinedload(models.Extract.text_fragment_identifiers)
+                                              ).order_by(models.Extract.creation_date
+                                                         ).all()
```

Comment:
I don't like this change.

Reply:
It's an auto-pep8 service that does this on another dev's machine.

Reply:
You should configure a pep8 config to not allow this indentation. There is no issue with flake8.